ui/qt-lib/src/Ariadne/UI/Qt/Widgets/Dialogs/NewWallet.hs

```diff
@@ -163,16 +117,13 @@ initNewWallet = do
   QWidget.setWindowTitle newWallet ("Create new wallet" :: String)
   QWidget.adjustSize newWallet
 
-  layout <- QVBoxLayout.new
-  QWidget.setLayout newWallet layout
-  QBoxLayout.setSpacing layout 18
-  QLayout.setContentsMarginsRaw layout 24 24 24 24
+  layout <- createLayout newWallet
 
   modeSelector <- createModeSelector
 
-  QBoxLayout.addWidget layout modeSelector
+  addHeader layout modeSelector
   void $ QLayout.setWidgetAlignment layout modeSelector $ alignHCenter .|. alignVCenter
-  QBoxLayout.addSpacing layout 42
+  void $ setProperty modeSelector ("styleRole" :: Text) ("dialogHeader" :: Text)
```

Comment:
I changed this 42 spacing into 24 (in `Util.hs`). All other dialogs in designs have 24 px spacing, I believe 42px in new wallet is an error or just some oddity.